modular_skyrat/modules/modular_items/lewd_items/code/verbs.dm

```diff
@@ -55,3 +55,57 @@
 		dropItemToGround(equipped_item, TRUE)
 
 	return TRUE
+
+/mob/living/carbon/human/verb/lick(mob/living/carbon/human/target in get_human_hearers_in_view(1, src))
+	set name = "Lick"
+	set category = "IC"
+
+	if(!istype(target))
+		return FALSE
+
+	var/taste = target.get_taste_or_smell()
+	if(!taste)
+		to_chat(src, span_warning("[target] doesn't seem to have a taste."))
+		return FALSE
+
+	to_chat(src, span_notice("[target] tastes like [taste]."))
+	to_chat(target, span_notice("[target] licks you."))
+
+/mob/living/carbon/human/verb/smell(mob/living/carbon/human/target in get_human_hearers_in_view(1, src))
+	set name = "Smell"
+	set category = "IC"
+
+	if(!istype(target))
+		return FALSE
+
+	var/smell = target.get_taste_or_smell(TRUE)
+	if(!smell)
+		to_chat(src, span_warning("[target] doesn't seem to have a smell."))
+		return FALSE
+
+	to_chat(src, span_notice("[target] smells like [smell]."))
+
+/// Returns the smell/taste of the parent mob
+/mob/living/carbon/human/proc/get_taste_or_smell(smell = FALSE)
```

Comment:
I think, in the code, this is akin to calling a proc with a magical number.
In my opinion it should either be SMELL and TASTE defines as the argument, or this should be wrapped in two separate procs.